### DISCUSSION TITLE
官网源码

### DISCUSSION THREAD

**Discussion by zuihou** [vuepress-theme-hope/vuepress-theme-hope#2939] [Q&A] [answered]
https://vuepress-theme-hope.github.io

请问这个网站的源码有没有开源？ 想拿来参考写用法

**Comment by Mister-Hope** [comment, marked as answer]
https://github.com/vuepress-theme-hope/vuepress-theme-hope.github.io

**Comment by zuihou** [comment]
抱歉，地址复制错了，我想问的是这个网站的源码：

https://theme-hope.vuejs.press/zh/guide/get-started/intro.html

**Reply by Mister-Hope** [reply]
左下角的编辑此页没看到？

**Reply by zuihou** [reply]
找到了

**Comment by zuihou** [comment]
顺便提个bug

![image](https://user-images.githubusercontent.com/8500410/226810015-aa399d09-03dd-42d3-8411-9de1655d3380.png)

**Reply by Mister-Hope** [reply]
![image](https://user-images.githubusercontent.com/33315834/226840749-ae67d21d-3102-41b9-a7b0-e447f87fc5cf.png)

可以参见 https://github.com/vuepress/vuepress-next/issues/1222 https://github.com/vuepress/vuepress-next/issues/901

**Comment by zuihou** [comment]
![image](https://user-images.githubusercontent.com/8500410/226823041-05b77bb6-755e-4c1a-a74e-33ff7eb171b7.png)

![image](https://user-images.githubusercontent.com/8500410/226823042-b7688385-be84-45d8-b6f0-036e4fe199f0.png)


我将 theme 文件夹单独复制出去后， docs-shared 的版本号是多少？

**Reply by zuihou** [reply]
![image](https://user-images.githubusercontent.com/8500410/226848824-fc458b76-4b2f-47b2-b4de-5e5125f5e7b0.png)


我写成 **2.0.0-beta.138** ，install 报错

**Reply by Mister-Hope** [reply]
doc-shared is a private package
